examples/matrix_multiplier/tests/test_matrix_multiplier.py

```diff
@@ -0,0 +1,181 @@
+import os
+from random import getrandbits
+
```

Comment:
See https://github.com/cocotb/cocotb/pull/1286#discussion_r364016538 for some changes suggested to this file

Reply:
Nevermind, you already did this. I think I never wore the comment I'm thinking of.